deployment/helm/resource-management-policies/balloons/templates/configmap_enable_nri.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if .Values.nri.enable }}
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Do we really need this configmap business, the script is not that super complex. I think it would be simpler and safer (no way to affect what get's executed in privileged mode in host pid namespace by just mangling a configmap).

I'd suggest to write the script in the init container spec, smth like
```yaml
    command:
      - "/bin/bassh"
      - "-c"
    args:
      - |
        echo Patching containerd config
        sed '/\[plugins.\"io.containerd.nri.v1.nri/,/^$/s/disable = false/disable = true/' -i /etc/containerd/config.toml
        systemctl restart containerd
```

Reply:
I'm personally not fan of adding multi line commands into pod spec especially when the command is long like with `sed` here. It makes it hard to read the DaemonSet file.

Reply:
I fail to see why the configmap is any more readable. What about the security concerns?

Reply:
I mean you should be able to put the script in the pod spec formatted exactly like in the configmap

Reply:
> I fail to see why the configmap is any more readable. What about the security concerns?

what kind of security concerns do you mean?

Reply:
> I mean you should be able to put the script in the pod spec formatted exactly like in the configmap

Yes I know. I just didn't like having multi line, long command in the spec of the pod. 

I tend to isolate things into their own entity, instead of having one big YAML. 
But anyways, I don't mean to say it is incorrect, it is just my personal preference and I'm totally fine to do it in the spec.

Reply:
> what kind of security concerns do you mean?

The visibility and assurance about what acutally gets run in the privileged host-pid-ns container. This whole business of mounting "some" configmap into the container and running a shell script from there